tests/ops/functions/test_simplify.py

```diff
@@ -162,3 +162,24 @@ def qfunc():
         assert s_op.data == simplified_tape_op.data
         assert s_op.wires == simplified_tape_op.wires
         assert s_op.arithmetic_depth == simplified_tape_op.arithmetic_depth
+
+    @pytest.mark.jax
+    def test_jitting_simplified_qfunc(self):
+        """Test that we can jit qnodes that have a simplified quantum function."""
```

Comment:
Should we also add a test case when we have multiple measurements to assert we get a tuple out? Or is this something we still don't support with jax?

Reply:
Maybe we should add a story for it somewhere? or a comment in return types documents to remember it?